### ISSUE TITLE
VSCode: [Linux] There is a Tooltip "Writing login information to the keychain failed ..." with Debian 11 Bullseye

### ISSUE THREAD

**Issue by MMtwo** [microsoft/vscode#148033]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.66.2
- OS Version: Debian 11 Bullseye

Steps to Reproduce:

1. open the VSCode
`code .`

**Actual Result**

![image](https://user-images.githubusercontent.com/70988862/165053865-ccd21553-d3c7-44d5-a0c7-7ce9c2254d90.png)

**Error Log**

Writing login information to the keychain failed with error 'GDBus, Error: org. freedesktop DBus, Error. ServiceUnknown: The name org. freedesktop.secrets was not provided by any .service files'.

**Comment by TylerLeonhardt** [comment]
/duplicate https://github.com/microsoft/vscode/issues/146553#issuecomment-1099549404

**Comment by JessMTermini** [comment]
I have the same exact issue. Started after I had to do a clean install on a reformat of my linux partition on my chromebook running chrome os with developer linux turned on. Everything was working fine prior. I've tried deleting code directory and .vscode in order to start over. I have tried reformatting my partition again and starting from scratch. I have tried merging, using local, use remote, and manual merge for options on startup, neither changes anything. I have tried logging out of github and then trying to force it to ask me to input my password thinking this might update/clear it. Nothing is working. It is driving me crazy, have spent hours on this now and am just short of losing my mind. I have so much work I am in the middle of right now and I can't get anything done everything is on hold. PLEASE SOMEBODY HELP!!!!!!!

**Comment by JessMTermini** [comment]
I'm about to do my 3rd reformat/reinstall. After which I will be forced to use something else cause I have NO IDEA how to even begin to fix this. Thx. :(

**Comment by TylerLeonhardt** [comment]
@JessMTermini can you please open a new issue with a screenshot/gif so I can see what you're seeing?

**Comment by JessMTermini** [comment]
Sure, I deleted my linux partition and haven't created a new one yet. I found [this article](https://rtfm.co.ua/en/what-is-linux-keyring-gnome-keyring-secret-service-and-d-bus/) and am reading this before I create a new one to hopefully help me be able to fix this. (No affiliation, just randomly found it on a google search) I have no idea if it will help me, but I figure hey, I always wanted to create a linux distro (ha! lol - I don't know if that will ever happen! lmao!!) so I figure what the heck, can't hurt me to know this stuff anyway, might as well learn it now and add it to my knowledge base! As soon as I create the new linux partition and it happens again I will definitely start a new issue. Thank you kindly for responding!!
